config/manifests/bases/hw-event-proxy-operator.clusterserviceversion.yaml

```diff
@@ -28,7 +28,7 @@ metadata:
     provider: Red Hat
     repository: https://github.com/openshift/hw-event-proxy
     support: Red Hat
-  name: bare-metal-event-relay-operator.v0.0.0
+  name: hw-event-proxy-operator.v0.0.0
```

Comment:
if we are going to use the d/s name we might want to change this back to `bare-metal-event-relay` else this is good

Reply:
this was the change that broke the `make bundle`. I have to change back to use upstream name here to be consistent with the package name.

Reply:
Is it possible to change the other place instead? We need the user visible package name and operator name to be bare-metal-xxx. Ignore my comment if this does not have impact on the operator name we use in subs and seen in csv.

Reply:
@yliu127 this is replaced in the d/s automation to bare-metal-event-relay we can leave it as hw-event-proxy-operator here.